mozetl/sync/bookmark_validation.py

```diff
@@ -105,10 +105,10 @@ def load(spark, bucket, prefix, version, start_date):
 @click.command()
 @click.option('--start_date', required=True, help="Date to process")
 @click.option('--end_date', help="Optional end date to run until")
-@click.option('--bucket', default='net-mozaws-prod-us-west-2-pipeline-analysis')
-@click.option('--prefix', default='kit/sync')
+@click.option('--bucket', default='telemetry-parquet')
+@click.option('--prefix', default='sync')
 @click.option('--input_bucket', default='telemetry-parquet')
-@click.option('--input_prefix', default='sync_summary/v1')
+@click.option('--input_prefix', default='sync_summary/v2')
```

Comment:
Do we need to set `.option("mergeSchema", "true")` in `extract`, when we call `spark.read`, or will this do the right thing when backfilling older data (or do we not need to worry about that)?

Reply:
The dataset isn't backward compatible because the column names don't match. You would have to use the script from the old revision to backfill everything before 2017-05-05(?).